src/Tribe/Cost_Utils.php

```diff
@@ -63,6 +63,10 @@ public function get_event_costs( $event ) {
 
 		$costs = tribe_get_event_meta( $event->ID, '_EventCost', false );
 
+		$costs = array_filter( (array) $costs, static function ( $cost ) {
+			return $cost !== '';
```

Comment:
:no_entry_sign: Use Yoda Condition checks, you must.

<sub>hash: c19487912df68f46ed99c7c36c91c204</sub>